erpnext/healthcare/doctype/patient/patient.py

```diff
@@ -58,7 +58,14 @@ def get_age(self):
 		if self.dob:
 			born = getdate(self.dob)
 			age = dateutil.relativedelta.relativedelta(getdate(), born)
-			age_str = str(age.years) + " year(s) " + str(age.months) + " month(s) " + str(age.days) + " day(s)"
+			age_str = "{years} {years_label} {months} {months_label} {days} {days_label}".format(
+				years = age.years,
+				years_label = _("Years(s)"),
+				months = age.months,
+				months_label = _("Month(s)"),
+				days = age.days,
+				days_label = _("Day(s)")
+			)
```

Comment:
```suggestion
			age_str = str(age.years) + _("Year(s)") + str(age.months) + _("Month(s)") + str(age.days) + _("Day(s)")
```
Check the uniform translation syntax used throughout the codebase [here](https://frappeframework.com/docs/user/en/translations).